README.md

```diff
@@ -34,6 +33,17 @@ Commonly used Gulp tasks shared across projects.
 - `gulp fractal:start`: Start Fractal server (if config path exists in `package.json`)
 - `gulp fractal:build`: Build static version of Fractal project
 
+## Options for js/css tasks
+
+Option | Effect
+:--- | :---
+`--minify` | Render additional minified files, suffixed with `.min`. Will be `true` in production builds.
+`--sourcemaps` | Render sourcemaps.
+`--fail-after-error` | See the 'Continuous Integration' section for details.
+`--concat` | By default, assets are compiled into individual files. Use this flag to concatenate them into a single js/css file.
```

Comment:
```suggestion
`--concat` | Concatenate CSS/JS asset files into a single aggregate file.
```

Reply:
@fabianmarz @wrux @stefanpl Wondering whether we should add an "Upgrading to v3" chapter to the readme (or changelog?) to explain how the invocation needs to be changed in order to keep the previous behavior?

Reply:
@sun That would help. Also I think we should list the default values for flags when running `build` or `build:production`. It's not clear without looking into the code if `--fail-after-error` is enabled in production.